javaForms/docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
     ]
 
   weston:
-    image: torizon/weston${GPU}:3
+    image: torizon/weston${GPU}:2
```

Comment:
using the bullseye on the app and bookworm on weston works?

Reply:
Yes, and for now it is necessary because there is a bug on the `weston-vivante:3` that makes this template and the .NET Avalonia GTK template to not work. But in the future, one this bug is fixed it can be moved back to 3.